### DISCUSSION TITLE
Output format

### DISCUSSION THREAD

**Discussion by technosaby** [technosaby/gsoc2022#17] [General]
At the initial stage I plan to dump the timestamps and the audio effects for a particular video.
 
- Is there a RedHen format ?
- Is there any general locations where output files will be placed ?

@turnermarkb @brucearctor

**Comment by brucearctor** [comment]
For very initial -- suggest to work with even a single file, and do things locally in some fashion.  Easy enough to modify from there.  Verify extraction.  Feel free to create a new format.  Once something exists, we have something tangible to improve upon.  Don't be concerned about "final" formats or locations at this point.

**Reply by technosaby** [reply]
@brucearctor I tried to use one radio hen video (news broadcast) and run through the pertained model. I received this. I think high level it is working. I will dump it in a new format to understand the data more closely. 

![temp](https://user-images.githubusercontent.com/89327872/173878474-d6ec73cc-f077-4a58-aaa9-07f50c287ee8.jpg)

**Comment by technosaby** [comment]
From Mark -- 

Please study https://www.redhenlab.org/home/the-cognitive-core-research-topics-in-red-hen/red-hen-data-format.
Not sure what you mean by timestamps. Timestamps are the most crucial part of the Red Hen data format:
they are used throughout the system to tie together data and metadata.
Look at the files in the tv tree on gallina.
/mnt/rds/redhen/gallina/tv

redhenlab.orgredhenlab.org
[Red Hen Lab GSoC 2022 Projects - Red Hen data format](https://www.redhenlab.org/home/the-cognitive-core-research-topics-in-red-hen/red-hen-data-format)
Red Hen aims to facilitate collaborative work across different types of expertise, geographical locations, and time. To facilitate this, we have developed a shared data format, specified below. To simplify interoperability and ensure we can scale, we currently rely on flat files rather than
[8:38](https://redhenlabs.slack.com/archives/C03K1AVL7R6/p1655262516680169)
For example, the .ocr files contain optical character recognition metadata. The .tpt files contain transcripts. The .seg files contain various different kinds of metadata.  They are all coordinated by timestamps.   Study the related files for a particular asset and you sill see how. You might think of creating a new extension, e.g. .sfx, for sound effects.

**Comment by technosaby** [comment]
From Peter Uhrig ---

Mark has already said everything important. Let me just add that you may also want to write ELAN EAF files, because this way you can load the video and the EAF into ELAN (https://archive.mpi.nl/tla/elan) to easily verify whether your annotations are good.

**Comment by technosaby** [comment]
Please review code from this [PR](https://github.com/technosaby/gsoc2022/pull/26)
